### PR TITLE
Standardizes area paths and area-based nomenclature

### PR DESCRIPTION
**Pull request by Son-of-Space**
## About The Pull Request

This PR repaths areas to follow a department - purpose - room formula (just department - room if purpose is unnecessary), reorganizes the area file to properly sort by department, removes some unused areas, and better utilizes child subtypes for areas. It fixes several problems with areas which were previously improperly subtyped causing problems like the lack of AI monitoring on vault areas and the inability to use the civilian door remote on most service areas.

## Why It's Good For The Game

Properly organizing our areas will make mistakes in adding areas or duplicates harder to do. By organizing each department in this way, it also provides better infrastructure for autoname devices and any future code that has to reference areas by department.

## Changelog
:cl:
fix: The civilian remote now properly affects service department doors
fix: Some areas are now properly affected by ai monitoring
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Please post the regex you used to preform this change, and also revert your changes to the changelogs. Also don't spellcheck stuff that's different in british/american english, it leads to language arguments, just leave it how you found it.

**Comment by Son-of-Space** [comment]
> 
> 
> Please post the regex you used to preform this change, and also revert your changes to the changelogs. Also don't spellcheck stuff that's different in british/american english, it leads to language arguments, just leave it how you found it.

I used VSCode's Replace in Files feature to make the change.

The reason why I had changed spellings of theatre to theater were because some used one version and some the other. To prevent people from making errors when referencing those, I thought it better to just have them all be spelled one way.

**Comment by Son-of-Space** [comment]
Let's see if I accidentally break something

File: html/changelogs/.all_changelog.yml [hunk 1]
```diff
@@ -1188,12 +1188,12 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
...
   - unknown: "Jesus christ it's a new map what the fuck\n\t"
-  - unknown: "Just kidding, it's only minor changes to medbay/mechbay/cybernetics/R&amp;D/toxins/robotics/chapel/theatre\n\
+  - unknown: "Just kidding, it's only minor changes to medbay/mechbay/cybernetics/R&amp;D/toxins/robotics/chapel/Theater\n\
       \t"
   - unknown: "Okay so there's too many changes to list completely, but basically:\
       \ toxins/R&amp;D/virology/xenobiology are south <i>through</i> medbay; robotics/cybernetics/mechbay\
-      \ are where toxins used to be, the theatre and chapel are above medbay.\n\t"
-  - unknown: "Theatre is a new place for the Clown and Mime to play, there are some\
+      \ are where toxins used to be, the Theater and chapel are above medbay.\n\t"
+  - unknown: "Theater is a new place for the Clown and Mime to play, there are some\
```
Comment (LemonInTheDark) [inline html/changelogs/.all_changelog.yml:1196]: You're still touching changelogs

Comment (Son-of-Space) [reply html/changelogs/.all_changelog.yml:1196]: AAAAAAAAAAAAAAAAAA

**Comment by Son-of-Space** [comment]
Today I learned through fire and brimstone how to properly revert changes.

File: code/modules/admin/verbs/secrets.dm [hunk 2]
```diff
@@ -335,7 +335,7 @@
 				return
 			SSblackbox.record_feedback("nested tally", "admin_secrets_fun_used", 1, list("Egalitarian Station"))
 			for(var/obj/machinery/door/airlock/W in GLOB.machines)
-				if(is_station_level(W.z) && !istype(get_area(W), /area/bridge) && !istype(get_area(W), /area/crew_quarters) && !istype(get_area(W), /area/security/prison))
+				if(is_station_level(W.z) && !istype(get_area(W), /area/command) && !istype(get_area(W), /area/commons) && !istype(get_area(W), /area/service) && !istype(get_area(W), /area/command/heads_quarters) && !istype(get_area(W), /area/security/prison))
```
Comment (LemonInTheDark) [inline code/modules/admin/verbs/secrets.dm:338]: Why the change here?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
I've got all of one question, and then we should be good for a merge

File: code/modules/admin/verbs/secrets.dm (hunk 2 above)
Comment (Son-of-Space) [reply code/modules/admin/verbs/secrets.dm:338]: This code targeted all of "crew quarters" for its things. Crew quarters, for some reason, put command staff quarters, common areas, and the service department all under one parent. To make sure it still targets all the same areas, I separated them into their respective new area paths.

Comment (LemonInTheDark) [reply code/modules/admin/verbs/secrets.dm:338]: aight sounds good, just wanted to check

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is now above zero (0). Feel free to make Feature/Tweak/Balance PRs.
